clients/include/testing_logging.hpp

```diff
@@ -0,0 +1,306 @@
+/* ************************************************************************
+ * Copyright 2016 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+#include <stdlib.h>
+#include <vector>
+
+#include "rocblas.hpp"
+#include "utility.h"
+#include "cblas_interface.h"
+#include "norm.h"
+#include "unit.h"
+#include "arg_check.h"
+#include <complex.h>
+#include <unistd.h>
+#include <pwd.h>
+#include <fstream>
+#include <string>
+#include <algorithm>
+#include <iterator>
+
+using namespace std;
+
+// replaces X in string with s, d, c, z or h depending on typename T
```

Comment:
AXPY and AMAX have natural X's in the name.  You may need to find a unique character glyph to replace

Reply:
the function names are lower case, so it is rocblas_Xaxpy and rocblas_Xamax. Only the upper case X is replaced, not the lower case x.